src/analyze/analyze-capability.c

```diff
@@ -5,10 +5,22 @@
 #include "cap-list.h"
 #include "capability-util.h"
 #include "format-table.h"
+#include "parse-util.h"
+
+static int table_add_capability(Table *table, int c)
+{
```

Comment:
coding style: please `{` on same line as function name, i.e. remove the line break here before the `{`